docs/HELM.md

```diff
@@ -0,0 +1,79 @@
+# Deploying the LogDNA Agent on Kubernetes using Helm
+
+[Helm][helm] is a package manager for Kubernetes, you can use it to deploy the LogDNA Agent in your Kubernetes
+cluster. 
+
+## Prerequisites
+
+- [Helm 3+][helm-install]
+- Kubernetes 1.13+
+
+## Installing the Chart
+
+Follow directions from https://app.logdna.com/pages/add-source to obtain your LogDNA Ingestion Key.
+
+To install the agent with the [release name][helm-concepts] `my-release` in your cluster use:
+
+```bash
+$ helm repo add logdna https://assets.logdna.com/charts
+$ helm install --set logdna.key=$LOGDNA_INGESTION_KEY my-release logdna/agent
+```
+
+You should see logs in https://app.logdna.com in a few seconds.
+
+### Tags support:
+
+Optionally, you can configure the LogDNA Agent to associate tags to all log records that it collects so that you can
+identify the agent's data quicker in the LogDNA web UI.
+
+```bash
+$ helm install --set logdna.key=$LOGDNA_INGESTION_KEY,logdna.tags=production my-release logdna/agent
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the LogDNA Agent chart and their default values. Note
+that only the ingestion key is required.
+
+Parameter | Description | Default
+--- | --- | ---
+`daemonset.tolerations` | List of node taints to tolerate | `[]`
+`daemonset.updateStrategy` | Optionally set an update strategy on the daemonset. | None
+`image.pullPolicy` | Image pull policy | `IfNotPresent`
+`logdna.key` | LogDNA Ingestion Key (Required) | None
+`logdna.tags` | Optional tags such as `production` | None
+`priorityClassName` | (Optional) Set a PriorityClass on the Daemonset | `""`
+`resources.limits.memory` | Memory resource limits | 75Mi
+`updateOnSecretChange` | Optionally set annotation on daemonset to cause deploy when secret changes | None
+`extraEnv` | Additional environment variables | `{}`
+`extraVolumeMounts` | Additional Volume mounts | `[]`
+`extraVolumes` | Additional Volumes | `[]`
+`serviceAccount.create` | Whether to create a service account for this release | `true`
+`serviceAccount.name` | The name of the service account. Defaults to `logdna-agent` unless `serviceAccount.create=false` in which case it defaults to `default` | None
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install --set logdna.key=LOGDNA_INGESTION_KEY,logdna.tags=production my-release logdna/agent
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the
+chart. For example,
+
```

Comment:
To avoid "passive voice" consider this rewrite:
"Alternatively, create a YAML file called `values.yaml` that specifies the values for the above parameters.  This file is then accessed and read during the chart installation."